### ISSUE TITLE
I have to install a new certificate every time I execute. Is there any way to solve this?

### ISSUE THREAD

**Issue by bbhxwl** [justcoding121/Titanium-Web-Proxy#707]
Every time I have to execute the following four lines of code, and then copy the CER file to IOS installation, and then I can use it. What is the way to install the certificate only once, and it doesn't matter if the program is restarted?

I am Chinese, English is not good. I can understand the code. I hope a master can give me a code

proxyServer.CertificateManager.CreateRootCertificate(false);
            proxyServer.CertificateManager.TrustRootCertificate();
            proxyServer.CertificateManager.TrustRootCertificateAsAdmin();
            File.WriteAllBytes("ca.cer", proxyServer.CertificateManager.RootCertificate.Export(System.Security.Cryptography.X509Certificates.X509ContentType.Cert));

**Comment by honfika** [comment]
Create the root certificae once with the CreateRootCertificate call.

Trust it manually (I quess the TrustRootCertificate is nto workin on IOS, maybe it is Windows only)

Export it to a byte array as you wrote, then save it anywere you want.

On the next start load it and set it with the `proxyServer.RootCertificate` property.

**Comment by bbhxwl** [comment]
> Create the root certificae once with the `CreateRootCertificate` call.
> 
> Trust it manually (I guess the TrustRootCertificate is not working on IOS, maybe it is Windows only)
> 
> Export it to a byte array as you wrote, then save it anywere you want.
> 
> On the next start load it and set it with the `proxyServer.RootCertificate` property.

I will, thank you, I want to know, if I don't modify the network data, can I let users do not need trust certificates? Like an HTTP proxy server. What's more, if I want the user to enter the account and password, my server can do the verification code?

**Comment by honfika** [comment]
I answered in the other ticket.

**Comment by honfika** [comment]
I close this ticket since it is similar your another ticket: #706
